webview.h

```diff
@@ -1213,7 +1221,11 @@ class webview : public browser_engine {
 } // namespace webview
 
 WEBVIEW_API webview_t webview_create(int debug, void *wnd) {
-  return new webview::webview(debug, wnd);
+  auto w = new webview::webview(debug, wnd);
```

Comment:
The `webview` instance leaks when returning null.

I would also suggest `nullptr` over `NULL` here. Checking for `!w->window()` would be fine too in my opinion.